protected/assets/js/Gp.js

```diff
@@ -0,0 +1,19 @@
+function removeSelectedContactLabel() {
+  $('#no_contact_label_result').hide();
+  $('.selected_contact_label span.name').text('');
+  $('#selected_contact_label_wrapper').hide();
+  $('#Contact_contact_label_id').val('-1');
+}
+
+function addItem(wrapper_id, ui){
```

Comment:
Could we just recycle the definition of this function in patient.js?
Prath defined a separate function, addGpItem, that handles adding a new GP as well. It would be worth consolidating them if we can.

Reply:
Good idea, but we'll have to make sure that patient.js purely just a library, and has no onLoad() callbacks or the like.

Reply:
Agreed.